docs/examples/identity/group.tf

```diff
@@ -0,0 +1,26 @@
+/*
+ * This example file shows how to create a group and add a user to it. 
+ */
+
+resource "oci_identity_group" "group1" {
+  name = "group1"
```

Comment:
-tf-group

Reply:
I was thinking about getting away from the -tf- prefix in examples since it might be unnecessarily inscrutable for casual example browsers.

Reply:
Yeah, I agree with that. Still, group1 isn't a great name. Ideally it should be clear to someone who finds this in the console later on where it came from. Maybe TerraformExampleGroup